package.json

```diff
@@ -31,7 +31,7 @@
     "test:watch": "jest --watchAll",
     "silent-test": "jest --silent",
     "test:bamboo": "jest --ci --reporters=default --reporters=jest-junit",
-    "lint": "eslint src"
+    "lint": "eslint ."
```

Comment:
this will now mean it is going to start running eslint on `babel.config` the jest config (I checked none seem to have any es-lint issues) but, I was just wondering if that is intended?

Reply:
Yeah that was intentional, everything should be checked, except the dist directory